package.json

```diff
@@ -124,6 +129,10 @@
 					"command": "datavirt.delete.datasource",
 					"when": "false"
 				},
+				{
+					"command": "datavirt.delete.datasource",
+					"when": "false"
+				},
```

Comment:
this looks a like a wrong copy. The command is duplicated

Reply:
Ouch...that is a copy/paste error. it should be datavirt.delete.vdb.